packages/manager/apps/telecom/src/app/telecom/telephony/repayments/repayments.html

```diff
@@ -0,0 +1,69 @@
+<section class="telephony-group-line-repayments">
+    <section class="widget-presentation">
+        <div class="row">
+            <div class="col-md-6">
+                <h4 data-translate="telephony_repayments_repayments"></h4>
+                <dl>
+                    <dt
+                        data-translate="telephony_repayments_pending_title"
+                    ></dt>
+                    <dd data-ng-bind="$ctrl.pendingAmountText"></dd>
+                    <dt
+                        data-translate="telephony_repayments_available_title"
+                    ></dt>
+                    <dd data-ng-bind="$ctrl.availableAmountText"></dd>
+                </dl>
+            </div>
+            <div class="col-md-6">
+                <h4 data-translate="telephony_repayments_costs"></h4>
+                <dl>
+                    <dt data-translate="telephony_repayments_costs_title"></dt>
+                    <dd data-ng-bind="$ctrl.costsAmountText"></dd>
+                </dl>
+            </div>
+        </div>
+
+        <oui-message
+            type="info"
+            class="mt-3"
+            style="display: inline-block !important"
```

Comment:
Did you tried using the bootstrap `inline-block` class?

Reply:
It's applied